app/org/sagebionetworks/bridge/play/controllers/StudyController.java

```diff
@@ -138,6 +140,15 @@ public Result createStudy() throws Exception {
         return okResult(new VersionHolder(study.getVersion()));
     }
 
+    public Result createSynapseAccount() throws Exception{
+        getAuthenticatedSession(DEVELOPER);
+
+        SynapseAccount newAccount = parseJson(request(), SynapseAccount.class);
+        String userId = studyService.createSynapseAccount(newAccount);
+
+        return createdResult(new SynapseUserIdHolder(userId));
+    }
```

Comment:
Since this doesn't affect studies, should this be in its own Controller and Service?